src/generator.ts

```diff
@@ -69,6 +69,9 @@ function generateFieldMiddlewareFromRule(
       if (res === true) {
         return await resolve(parent, args, ctx, info)
       } else if (res === false) {
+        if (typeof options.fallbackError === 'function') {
+          return await options.fallbackError(null, parent, args, ctx, info)
```

Comment:
@maticzav I think this breaks when you have a simple rule that returns `false`. `null` isn't thrown, so the example isn't accurate, and this wouldn't be an internal server error.

Any recommendations on how to handle this?

Reply:
Could you open a PR with the fix?

Reply:
@maticzav Would be happy to try! But I'm unsure what the best approach is here. Should we remove this clause altogether? What would false-returning rules default to, if the fallback error is only handling thrown situations? Or should we change the docs of what the `fallbackError` handles to include non-thrown errors?

Reply:
```
        if (typeof options.fallbackError === 'function') {
          return await options.fallbackError(null, parent, args, ctx, info)
```

Our code actually relies on this so we would rather prefer to keep this 😅

Reply:
Oh, wait actually I don't understand anymore why this clause is here 😅 It seems our code actually only relies on the `fallbackError` call down below.